infra/src/test/kotlin/fi/fta/geoviite/infra/inframodel/InfraModelParsingIT.kt

```diff
@@ -33,20 +29,27 @@ const val TESTFILE_CLOTHOID_AND_PARABOLA = "/inframodel/testfile_clothoid_and_pa
 @ActiveProfiles("dev", "test")
 @SpringBootTest
 class InfraModelParsingIT @Autowired constructor(
-    codeDictionaryService: CodeDictionaryService,
     geographyService: GeographyService,
-    val heightTriangleDao: HeightTriangleDao,
     switchStructureDao: SwitchStructureDao,
     val trackNumberDao: LayoutTrackNumberDao,
     val kkJtoETRSTriangulationDao: KKJtoETRSTriangulationDao,
 ): ITTestBase() {
-    private val mapper = jacksonObjectMapper()
-    private val featureTypes: List<FeatureType> = codeDictionaryService.getFeatureTypes()
     private val coordinateSystemNameToSrid = geographyService.getCoordinateSystemNameToSridMapping()
     private val switchStructuresByType = switchStructureDao.fetchSwitchStructures().associateBy { it.type }
-    private val switchStructuresById = switchStructureDao.fetchSwitchStructures().associateBy { it.id as IntId }
     private val switchTypeNameAliases = switchStructureDao.getInframodelAliases()
 
+    @Test
+    fun importingBOMFileWithISOXmlEncodingWorks() {
```

Comment:
Mainiot testit. Tämän voisi monistaa myös koko sallittujen merkistöjen joukolle.

Reply:
Hyvä idea. Alunperin tämä oli täällä vähän niinkuin happotestinä tälle koko toiminnallisuudelle, mutta näitä voisi tosiaan monistaa useammallekin merkistölle (esim. otetaan huomioon eri UTF-BOM:it jne.)